packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -23,3 +23,5 @@ export const TELEMETRY_OPT_OUT_LINK =
   'https://github.com/forcedotcom/salesforcedx-vscode/wiki/Telemetry';
 export const PUSH_OR_DEPLOY_ON_SAVE_ENABLED = 'push-or-deploy-on-save.enabled';
 export const RETRIEVE_TEST_CODE_COVERAGE = 'retrieve-test-code-coverage';
+export const INITIAL_SOBJECT_REFRESH_ON_STARTUP =
```

Comment:
I'd suggest we change the name of this flag to `enable-sobject-refresh-on-startup` or something like that because the new vscode settings UI makes this type of flag format look like the setting is already enabled.

Reply:
So instead of `initial-sobject-refresh-on-startup.enabled` make it `enable-sobject-refresh-on-startup`?